### ISSUE TITLE
API not found after following examples

### ISSUE THREAD

**Issue by andjosh** [heroku/heroku-kong#14]
I feel like I must be missing something here, so hopefully there's an easy point I've skipped.

I followed the [example to set up `kong-admin` as a proxy](https://github.com/heroku/heroku-kong#authenticated-admin-api) to the letter. When making requests to check results through the Heroku bash session, everything works perfectly:

~~~sh
# heroku run bash
# $ kong start -c $KONG_CONF
curl -i -X GET http://localhost:$PORT/kong-admin/apis
HTTP/1.1 200 Success
Date: Tue, 02 Aug 2016 21:53:27 GMT
Content-Type: application/json; charset=utf-8
Transfer-Encoding: chunked
Connection: keep-alive
Access-Control-Allow-Origin: *
Server: kong/0.7.0

{"data":[{"upstream_url":"http:\/\/localhost:8001\/","request_path":"\/kong-admin","id":"f87dcde1-e6b0-46d4-b25b-dd31ef6e517a","strip_request_path":true,"name":"kong-admin","created_at":1470174925000}],"total":1}
~~~

But, when requesting the path externally, I get the response:

~~~sh
# from my local machine (external to the Heroku session)
curl --url http://<heroku app>.herokuapp.com/kong-admin/apis --header 'apiKey: <my api key>'

{"request_path":"\/kong-admin\/apis","message":"API not found with these values","request_host":["<heroku app>.herokuapp.com"]}
~~~

I get the same behavior (API not found) when following the simpler [example for a mock bin proxy](https://getkong.org/docs/0.8.x/getting-started/adding-your-api/).

What am I missing here? Is there a configuration step where I expose.... something?

**Comment by andjosh** [comment]
So, after trying lots of avenues, I forced a rebuild of the app (`git push heroku -f`), and things seem to work. Closing the issue.

**Comment by mars** [comment]
Thanks for following up & closing this issue.

With this 0.7 version of Kong on Heroku, I have seen config changes fail to apply until a `heroku restart` or any thing else that causes a restart, like a new deploy.

Looking forward to this being fixed when we eventually release Kong 0.9 with Postgres.
